examples/hybridexampleapp/android/app/src/main/java/com/instabug/hybridsampleapp/MainActivity.java

```diff
@@ -0,0 +1,43 @@
+package com.instabug.hybridsampleapp;
+
+import android.content.Intent;
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import com.instabug.crash.CrashReporting;
+import com.instabug.crash.models.IBGNonFatalException;
+import com.instabug.hybridsampleapp.databinding.ActivityMainBinding;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        com.instabug.hybridsampleapp.databinding.ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
```

Comment:
I guess we can move `com.instabug.hybridsampleapp.databinding` to an import instead of using the full package name before the class name, what do you think?